repository/decorator/repository_decorator.go

```diff
@@ -10,6 +10,12 @@ import (
 	"github.com/serhiihuberniuk/blog-api/models"
 )
 
+const (
+	userObjectType    = "user:*"
+	postObjectType    = "post:*"
+	commentObjectType = "comment:*"
```

Comment:
please use key prefixes without '*', just `user:<id>`

Reply:
please name constants with the same prefix, `objectTypeUser`, `objectTypePost`...